src/main/java/br/com/mercadolivre/bootcampw2/grupo11/socialmeli/entities/Product.java

```diff
@@ -0,0 +1,28 @@
+package br.com.mercadolivre.bootcampw2.grupo11.socialmeli.entities;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@Entity
+public class Product {
+    @Id
+    @Column(name = "product_id")
+    private Long productId;
```

Comment:
Is redundant to call productId, we are already inside Product, just call it 'id'